File: source-code/Server/backend/src/main/java/com/iasdietideals24/backend/mapstruct/dto/OffertaSilenziosaDto.java
package com.iasdietideals24.backend.mapstruct.dto;

import com.iasdietideals24.backend.mapstruct.dto.shallows.AstaShallowDto;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class OffertaSilenziosaDto extends OffertaDiCompratoreDto {

    private Boolean isAccettata = null;

    private AstaShallowDto astaRiferimentoShallow;
}

File: source-code/Client/frontendandroid/app/src/main/java/com/iasdietideals24/dietideals24/utilities/dto/OffertaSilenziosaDto.kt
package com.iasdietideals24.dietideals24.utilities.dto

import com.iasdietideals24.dietideals24.utilities.dto.shallows.AccountShallowDto
import com.iasdietideals24.dietideals24.utilities.dto.shallows.AstaShallowDto
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import java.math.BigDecimal
import java.time.LocalDate
import java.time.LocalTime

@Getter
@Setter
@NoArgsConstructor
class OffertaSilenziosaDto {
    private val idOfferta: Long? = null

    private val dataInvio: LocalDate? = null

    private val oraInvio: LocalTime? = null

    private val valore: BigDecimal? = null

    private val compratoreCollegatoShallow: AccountShallowDto? = null

    private val isAccettata: Boolean? = null

    private val astaRiferimentoShallow: AstaShallowDto? = null
}
